Godeps/_workspace/src/k8s.io/kubernetes/pkg/kubectl/describe.go

```diff
@@ -602,6 +603,7 @@ func describePod(pod *api.Pod, events *api.EventList) (string, error) {
 	return tabbedString(func(out io.Writer) error {
 		fmt.Fprintf(out, "Name:\t%s\n", pod.Name)
 		fmt.Fprintf(out, "Namespace:\t%s\n", pod.Namespace)
+		fmt.Fprintf(out, "SCC:\t%s\n", pod.Annotations[allocator.ValidatedSCCAnnotation])
```

Comment:
When running against external kube, or without SCC admission, this could be blank